src/components/structures/InteractiveAuth.js

```diff
@@ -141,16 +141,20 @@ export default React.createClass({
     },
 
     _requestCallback: function(auth, background) {
-        this.setState({
-            busy: !background,
-            errorText: null,
-            stageErrorText: null,
-        });
+        // only set the busy flag if this is a non-background request
```

Comment:
well, I'm still not grokking it. A comment that says "do X" doesn't help me understand why X is the right thing to do :/